provisioning/virtual-machines/setup-crownlabs-vm/ansible/xubuntu-ansible-crownlabs-playbook.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: all
+  become: yes
+
+  vars:
+    crownlabs_mode: True
+
+  roles:
+  - xubuntu-pre
+  - crownlabs
```

Comment:
The installation of Ansible was part of the lab itself. However, I created a different file because I was not sure a vanilla VM was enough and extra configuration was necessary. If that is not the case, I can delete this file.